nemo/lightning/pytorch/accelerate/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
@oyilmaz-nvidia any objections to deprecating the accelerate directory?

Reply:
I'd rather keep it since there will be more files and possibly other model accelerators in this folder. Having a callback and the code for model acceleration should be separated IMO.